test/cli_test.js

```diff
@@ -36,6 +36,9 @@ describe('cli', function() {
   });
 
   describe('build', function() {
+    beforeEach(function() {
+      rimraf.sync('dist');
```

Comment:
We should generally not have to cleanup both before and after like this. Having to cleanup before generally indicates some other test is leaving us in an invalid starting state...